src/batch_triangular/Xgemm_batch_core.cuh

```diff
@@ -434,15 +434,12 @@ int Xgemm_batch_nonuniform_core(kblasHandle_t handle,
       }
       magmablas_Xgemm_vbatched_core((magma_trans_t)(MagmaNoTrans + (transA == KBLAS_Trans)),
                                     (magma_trans_t)(MagmaNoTrans + (transB == KBLAS_Trans)),
-                                    (magma_int_t*)m, (magma_int_t*)n, (magma_int_t*)k,
```

Comment:
Can you add an ifdef on the magma version to support order versions?

Reply:
@pghysels  ping. This would be also useful for PETSc https://gitlab.com/petsc/petsc/-/merge_requests/6206

Reply:
Sorry, I have missed you already did that. Thanks again, it was on my TODO list